### PR TITLE
Updates to slsutil.banner

### PR DESCRIPTION
**Pull request by amendlik**
### What does this PR do?
Address a number of issues with `slsutil.banner`
1. Typo in the default banner text
2. Issue when the comment character was padded with a space
3. More examples in the documentation
4. Unhandled exception when the width was too small

### What issues does this PR fix or reference?
None

### Tests written?
Yes

### Commits signed with GPG?
Yes

### CODE REVIEW

**Comment by amendlik** [comment]
@cmcmarrow We'll want to put this on the neon branch, now that that exists.

**Comment by amendlik** [comment]
@Ch3LL, @cmcmarrow Any update on this PR?

**Comment by Ch3LL** [comment]
@amendlik i went ahead and added it to be backported to neon once this is merged.

bump @cmcmarrow waiting on your review here.

**Comment by amendlik** [comment]
@Ch3LL, can you contact @cmcmarrow and let him know we're waiting on him? I had another PR that sat for 6 weeks and later found out the assignee had his GitHub notifications turned off. Maybe that is what's going on here.

File: salt/modules/slsutil.py
Comment (cmcmarrow) [inline salt/modules/slsutil.py]: Can we change 'ACME Corp' to Salt for this example?

**Review by cmcmarrow — CHANGES_REQUESTED** [review]

**Comment by cmcmarrow** [comment]
@amendlik I just have one request for change. Other than that it looks good. Thanks for the test!

File: salt/modules/slsutil.py
Comment (cmcmarrow) [reply salt/modules/slsutil.py]: Its because ACME Corp is a thing. https://www.acmecorp.com/

**Comment by Ch3LL** [comment]
ping  @cmcmarrow  this is ready for your re-review

**Comment by amendlik** [comment]
@cmcmarrow @Ch3LL What is going on at SaltStack that PR's now take 2 months to be merged? It used to be a week at the most. I don't mean to sound critical, but the level of support for contributors has dropped drastically over the last year, and SaltStack has done nothing to manage expectations.

**Comment by Ch3LL** [comment]
@amendlik we have been slowing down merges as we attempt to stabilize our test suite. There's a bit more explanation here: https://groups.google.com/forum/#!topic/salt-announce/5RJjtDYszx8

We want to lower the gap between merge and release time, but unfortunately that requires us to slow down on merges as we ensure our test suite is stabilized. I'll ping @cmcmarrow offline because it looks like we might be able to get this one in. Let me know if I can clarify anything else.

**Comment by s0undt3ch** [comment]
Master port https://github.com/saltstack/salt/pull/56761

**Pull request merged**
